### PR TITLE
ecore_con: Add guards to some includes

### PR DESCRIPTION
**Pull request by Coquinho**
Based on [946db](https://github.com/expertisesolutions/efl/commit/946db)

### CODE REVIEW

File: src/lib/ecore_con/Ecore_Con.h [hunk 1]
```diff
@@ -96,38 +101,38 @@
...
- * Eina_Strbuf APIs, abstract dynamic buffer management and make it trivial 
- * to handle buffers at runtime, without having to manage them. Eina_Binbuf 
- * makes it possible to create, expand, reset and slice a blob of memory - 
- * all via API. No system calls, no pointer manipulations and no size 
+ *
+ * To help with this Eina has some handy API's. The Eina_Binbuf and
+ * Eina_Strbuf APIs, abstract dynamic buffer management and make it trivial
+ * to handle buffers at runtime, without having to manage them. Eina_Binbuf
+ * makes it possible to create, expand, reset and slice a blob of memory -
+ * all via API. No system calls, no pointer manipulations and no size
```
Comment (Coquinho) [inline src/lib/ecore_con/Ecore_Con.h:114]: oh boy... @JPTIZ, are you gonna make me add trailing spaces again?

Comment (JPTIZ) [reply src/lib/ecore_con/Ecore_Con.h:114]: `git checkout -p -- <file>` and use `y/n/s` :^D

JK, maybe we can solve this later.

File: src/lib/ecore_con/ecore_con_url_curl.h [hunk 2]
```diff
@@ -8,7 +8,7 @@
  */
 #include <curl/curl.h>
 #else
-#ifdef __WIN32__
+#ifdef __WIN32
```
Comment (JPTIZ) [inline src/lib/ecore_con/ecore_con_url_curl.h:11]:
```suggestion
#ifdef _WIN32
```

Comment (joaoantoniocardoso) [reply src/lib/ecore_con/ecore_con_url_curl.h:11]: wow, now I'll suspect every single underline.

**Pull request merged**